packages/rainbowkit-wallets/src/components/CreateWalletModal.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react'
+import Modal from 'react-modal'
+
+const customStyles = {
+  content: {
+    top: '50%',
+    left: '50%',
+    right: 'auto',
+    bottom: 'auto',
+    marginRight: '-50%',
+    transform: 'translate(-50%, -50%)',
+    zIndex: 9999999999,
+  },
+  overlay: { zIndex: 9999999999 },
+}
+
+type ICreateWalletProps = {
+  onSubmit: (data: any) => void
+  onExit: () => void
+}
+
+export function CreateWalletModal(props: ICreateWalletProps): React.JSX.Element {
+  const [modalIsOpen, setIsOpen] = useState(true)
+  const [username, setUsername] = useState('')
+  const [password, setPassword] = useState('')
+
+  function closeModal() {
+    setIsOpen(false)
+    props.onExit()
+  }
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault()
+    setIsOpen(false)
+    props.onSubmit({ username, password })
+  }
+
+  return (
+    <div>
```

Comment:
There should be no need for this parent `<div>`. Also, if you want just to wrap components to export a single component, use fragments instead (`<></>`).

Reply:
Sure. isn't <></> the short syntax for React.Fragment (https://legacy.reactjs.org/docs/fragments.html#short-syntax)?

Reply:
That's why I said "use fragments instead" 😅

Reply:
not sure what I read xD